how-to/integrate-with-sso/integrate-with-okta/public/manifest.fin.json

```diff
@@ -20,22 +20,23 @@
 		"company": "OpenFin",
 		"description": "A way of showing examples of what OpenFin can do.",
 		"icon": "http://localhost:8080/favicon.ico",
-		"name": "Integrate with SSO - v10.0.0",
+		"name": "Integrate with Okta - v10.0.0",
 		"target": ["desktop", "start-menu"]
 	},
 	"supportInformation": {
 		"company": "OpenFin",
-		"product": "Workspace Starter - Integrate With SSO - Client",
+		"product": "Workspace Starter - Integrate With Okta - Client",
 		"email": "support@openfin.co",
 		"forwardErrorReports": true
 	},
 	"customSettings": {
-		"auth": {
-			"domain": "<DOMAIN>",
-			"clientId": "<CLIENTID>",
-			"loginUrl": "http://localhost:8080/platform/login-complete.html",
-			"logoutUrls": ["http://localhost:8080/platform/logout-complete.html", "https://accounts.google.com"],
+		"okta": {
+			"domain": "{YOUR_DOMAIN}}",
+			"clientId": "{YOUR_CLIENTID}",
+			"loginUrl": "{YOUR_LOGIN_URL}",
+			"logoutUrls": ["{YOUR_LOGOUT_URL}"],
 			"appUrl": "http://localhost:8080/platform/app.html",
+			"redirectUrl": "http://localhost:8080/platform/app.html",
```

Comment:
There should be no redirectUrl, that is the loginUrl, to follow the same pattern as the auth0 example